src/context.rs

```diff
@@ -84,6 +97,10 @@ impl ContextTrait for Context {
         serde_json::to_string(&self.services)
     }
 
+    fn websockets_iterator(&self) -> hash_map::Values<ws::util::Token, ws::Sender> {
```

Comment:
sorry to be nitpicking again: the trend in Rust land seems to be to just use the short `iter` form (eg. https://doc.rust-lang.org/std/collections/struct.HashMap.html#method.iter) so let's use websockets_iter(&self) here.